lib/ability.rb

```diff
@@ -60,6 +60,7 @@ def initialize(user, resource, controller = nil)
       can [:send_receipt, :show], Order
       if resource == Facility.cross_facility
         can :manage, [Reservation, User]
+        cannot [:edit, :update], Reservation
```

Comment:
Good call. It's looking like a BA may not need any explicit `Reservation` abilities, since the `FacilityUserReservationsController` authorizes against `OrderDetail`. Updating.